### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by taehoonlee**
This PR fixes several typos in the warning and the error messages.

### CODE REVIEW

File: tensorflow/compiler/mlir/tensorflow/translate/tf_mlir_translate.cc [hunk 1]
```diff
@@ -119,7 +119,7 @@ mlir::OwningModuleRef GraphdefToSplattedMlirTranslateFunction(
               break;
             default:
               inst.emitWarning()
-                  << "Skipping splat converstion for "
+                  << "Skipping splat conversation for "
```
Comment (mihaimaruseac) [inline tensorflow/compiler/mlir/tensorflow/translate/tf_mlir_translate.cc]: I think the word here should be `conversion`

**Review by mihaimaruseac — COMMENTED** [review]
Thank you for the fixes. There is one place where I think another word should be used, but all looks good.

**Comment by gbaned** [comment]
@taehoonlee Could you please address the reviewer comments and resolve conflicts? Thanks!

**Comment by taehoonlee** [comment]
Thank you for the comment, @mihaimaruseac. Updated.

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**